actor/5e_lay_on_hands.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * System: D&D5e
+ * Apply lay-on-hands feat to a target character.  Asks the player how many HP to heal and
+ * verifies the entered value is within range before marking down usage counter. If the player
+ * has OWNER permissions of target (such as GM or self-heal) the HP are applied automatically; 
+ * otherwise, a 'roll' message appears allowing the target character to right-click to apply healing.
+ */
+
+let confirmed = false;
+let actorData = actor ? actor : canvas.tokens.controlled[0];
```

Comment:
We could try the default character if no token is controlled:

```suggestion
let actorData = actor ? actor : canvas.tokens.controlled[0] || game.user.character;;
```